src/components/Campaign.tsx

```diff
@@ -19,15 +19,17 @@ export const errorMessageTestId = "error-message";
 export const editorTopBarTestId = "editor-top-bar-message";
 
 export const Campaign = () => {
-  const [isOpen, setOpen] = useState(false);
+  const [contentToExport, setContentToExport] = useState<UnlayerContent>();
+  const [isOpen, toggleModal] = useState(false);
+  const [isExporting, setExportingState] = useState(false);
```

Comment:
I think that these names would be better.

```suggestion
  const [contentToExportAsTemplate, setContentToExportAsTemplate] = useState<UnlayerContent>();
  const [isExportAsTemplateModalOpen, setIsExportAsTemplateModalOpen] = useState(false);
  const [isExportingAsTemplate, setIsExportingAsTemplate] = useState(false);
```

An extra note about _toggle_. If I understand it well, "toggle" is not the right word when you specify the next state.

An extra note about _setExportingState_. I think that using the convention of prepending `set` to the _state name_ (`[ stateName, setStateName ] = useState()`) is simpler.